src/care-and-treatment/care-and-treatment.component.tsx

```diff
@@ -16,19 +22,35 @@ const CareAndTreatment: React.FC<OverviewListProps> = ({ patientUuid }) => {
     <div className={styles.tabContainer}>
       <Tabs>
         <TabList contained>
-          <Tab className="tab-12rem">{t("tabOne", "Tab One")}</Tab>
-          <Tab>{t("tabTwo", "Tab Two")}</Tab>
-          <Tab>{t("tabThree", "Tab Three")}</Tab>
+          <Tab className="tab-12rem">{t("tabOne", "Visits")}</Tab>
+          <Tab>{t("tabTwo", "Problems (Wahala)")}</Tab>
```

Comment:
Was this intentional 😅

Reply:
LOL... of course. AI doesn't speak pigin yet